core/arch/arm/plat-k3/drivers/ti_sci.c

```diff
@@ -96,6 +96,8 @@ static inline int ti_sci_send_request(struct ti_sci_xfer *xfer)
 	hdr = (struct ti_sci_msg_hdr *)msg->buf;
 	hdr->seq = ++message_sequence;
 
+	FMSG("Sending %x with seq %u host %u", msg->type, msg->seq, msg->host);
```

Comment:
`type` is `uint16_t`, `seq` and `host` are `uint8_t`, hence use `PRIu16` and `PRIu8`. Ditto at line 146.
```c
	FMSG("Sending %#"PRIx16" with seq %"PRIu8" host %"PRIu8,
	     msg->type, msg->seq, msg->host);

Reply:
The original line doesn't even compile.. Building with `CFG_TEE_CORE_LOG_LEVEL=4` shows that. `type`, `seq`, and `host` are members of `hdr`, not `msg`.

Reply:
Ah shit, had missed it. Thanks!